### ISSUE TITLE
Multiple boxes and few points to remove certain region within certain box

### ISSUE THREAD

**Issue by KLiang-13** [facebookresearch/segment-anything#484]
![Input](https://github.com/facebookresearch/segment-anything/assets/87765044/ae79aeee-ff73-401a-9499-19cd2d60c0ef)
As you can see in the image above, I would like to segment the image above using 5 boxes as input. Within the 5 boxes, I will need to refine 1 of the box by specifying the points onto it. How can I merge the masks? Below are the codes I used to draw the boxes and points.

```
input_boxes = torch.tensor([
    coordinates[0],
    coordinates[1],
    coordinates[2],
    coordinates[3],
    coordinates[4],
], device=predictor.device)

transformed_boxes = predictor.transform.apply_boxes_torch(input_boxes, image.shape[:2])

# green / red points
input_point = np.array([[[875, 400]], [[1160, 400]], [[900, 630]], [[1145, 630]],[[0,0]]])
input_label = np.array([[1], [1], [0], [0], [-1]])

# Show result
plt.figure(figsize=(10, 10))
plt.imshow(image)

for point, label in zip(input_point, input_label):
    print(point, label)
    show_points(point, label, plt.gca())

input_point = torch.from_numpy(input_point).to(device=device)
input_label = torch.from_numpy(input_label).to(device=device)

masks, _, _ = predictor.predict_torch(
    point_coords=input_point,
    point_labels=input_label,
    boxes=transformed_boxes,
    multimask_output=False,
)
```

**Desired Output:**
![DesiredOutput](https://github.com/facebookresearch/segment-anything/assets/87765044/64db42a9-9cfd-4396-a655-6206d4256eef)

**Comment by swframe** [comment]
Could you explain what you mean by "merge the masks"? The mask are 2d arrays. 
Do you just want to logically-or them together.
```
total = np.zeros(masks[0].shape)
for mask in masks:
     total = np.logical_or(total > 0, mask > 0).astype(np.int8)
```

**Comment by KLiang-13** [comment]
> Could you explain what you mean by "merge the masks"? The mask are 2d arrays. Do you just want to logically-or them together.
> 
> ```
> total = np.zeros(masks[0].shape)
> for mask in masks:
>      total = np.logical_or(total > 0, mask > 0).astype(np.int8)
> ```

Yes, that's what I meant to say, thank you very much ^^
